packages/icons/deprecated.yml

```diff
@@ -1 +1,6 @@
 deprecated: []
+- name: Organization
+    subcategories:
+      - name: Systems
+        members:
+          - kubernetes
```

Comment:
it looks like ci doesn't like something about the formatting here, can you fix the formatting?
<img width="1048" alt="Screen Shot 2022-05-09 at 3 28 02 PM" src="https://user-images.githubusercontent.com/32556167/167492222-79e928dd-ccee-40e6-84e5-c2734678e13d.png">

Reply:
Based on [what this file previously used to look like](https://github.com/conradennis/carbon/commit/eda5c7de6591b1ddc36a31c2577f35e57c6b2ca5#diff-14365a39de10c1ccfde6b7ee38b7fa6173870479719eae6877adcc80ed49c6f8), it appears this file should be just names and reasons for deprecations.


```yml
deprecated:
  - name: app-switcher
    reason: The app switcher icon has been replaced by switcher
  - name: arrows
    reason: The arrows icon has been replaced by arrows--vertical
  # ...etc
```

Reply:
@jnm2377 sure, any insight on how I should format it? When I open the deprecated.yml file it just says:

Deprecated:[]

so should I put the icon name in the brackets and I'm good to go? Sorry not familiar with this!

Reply:
@jnm2377 thanks for the heads up
@tay1orjones thanks for sharing, I'll reformat as shown above.